os/tools/mkbinheader.py

```diff
@@ -175,7 +150,7 @@
     fp.write(struct.pack('I', file_size))
     fp.write('{:{}{}.{}}'.format(binary_name, '<', SIZE_OF_BINNAME, SIZE_OF_BINNAME - 1).replace(' ','\0'))
     fp.write('{:{}{}.{}}'.format(binary_ver, '<', SIZE_OF_BINVER, SIZE_OF_BINVER - 1).replace(' ','\0'))
-    fp.write(struct.pack('I', binary_ram_size))
+    fp.write(struct.pack('I', int(dynamic_ram_size)))
```

Comment:
I and Sunghan think the original is better. Because it is not easy to determine the `dynamic_ram_size` including static size.

Reply:
I have made the changes to properly calculate the RAM size in the script.